Sandbox/demo-scichart-angular/src/app/app.component.ts

```diff
@@ -0,0 +1,438 @@
+      pointMarker: new EllipsePointMarker(wasmContext, {
+        width: 64,
+        height: 64,
+        fill: appTheme.VividSkyBlue + '77',
+        strokeThickness: 0,
+      }),
+      dataSeries: new XyzDataSeries(wasmContext, { xValues, yValues, zValues }),
+    });
+    sciChartSurface.renderableSeries.add(bubbleSeries);
+
+    // Create a Line Series
+    const lineSeries = new SplineLineRenderableSeries(wasmContext, {
+      stroke: appTheme.VividSkyBlue,
+      strokeThickness: 2,
+      opacity: 0.7,
+      dataSeries: new XyDataSeries(wasmContext, { xValues, yValues }),
+    });
+    sciChartSurface.renderableSeries.add(lineSeries);
+
+    // Add watermark annotation
+    const watermark = new TextAnnotation({
+      text: '',
+      x1: 0.5,
+      y1: 0.5,
+      fontSize: 42,
+      opacity: 0.5,
+      xCoordinateMode: ECoordinateMode.Relative,
+      yCoordinateMode: ECoordinateMode.Relative,
+      horizontalAnchorPoint: EHorizontalAnchorPoint.Center,
+      verticalAnchorPoint: EVerticalAnchorPoint.Center,
+    });
+    sciChartSurface.annotations.add(watermark);
+
+    // Add title annotation
+    sciChartSurface.annotations.add(
+      new TextAnnotation({
+        text: 'Series Startup Animations in SciChart.js',
+        fontSize: 18,
+        textColor: appTheme.ForegroundColor,
+        x1: 0.5,
+        y1: 0,
+        opacity: 0.77,
+        horizontalAnchorPoint: EHorizontalAnchorPoint.Center,
+        xCoordinateMode: ECoordinateMode.Relative,
+        yCoordinateMode: ECoordinateMode.Relative,
+      })
+    );
+
+    // Loop forever and update animations
+    let animationState = 0;
+    const updateAnimation = () => {
+      let currentAnimation: SeriesAnimation = waveAnimation;
+      switch (animationState) {
+        case 0:
+          currentAnimation = waveAnimation;
+          sciChartSurface.addAnimation(
+            typeWriterAnimation(watermark, 'Wave Animation')
+          );
+          animationState++;
+          break;
+        case 1:
+          currentAnimation = sweepAnimation;
+          sciChartSurface.addAnimation(
+            typeWriterAnimation(watermark, 'Sweep Animation')
+          );
+          animationState++;
+          break;
+        case 2:
+          currentAnimation = scaleAnimation;
+          sciChartSurface.addAnimation(
+            typeWriterAnimation(watermark, 'Scale Animation')
+          );
+          animationState++;
+          break;
+        case 3:
+          currentAnimation = fadeAnimation;
+          sciChartSurface.addAnimation(
+            typeWriterAnimation(watermark, 'Fade Animation')
+          );
+          animationState = 0;
+          break;
+      }
+      lineSeries.enqueueAnimation(currentAnimation);
+      bubbleSeries.enqueueAnimation(currentAnimation);
+      // Loop forever while SciChartSurface is not deleted (see React Component unmount)
+      if (!sciChartSurface.isDeleted) setTimeout(updateAnimation, 2000);
```

Comment:
A better animation cleanup demonstration is required.
e.g. use lifecycle methods.